### DISCUSSION TITLE
Display more lines when using show context in explore

### DISCUSSION THREAD

**Discussion by yandingkang** [grafana/grafana#43259] [General]
When I use explore to display logs, after I click 'show context', I can only see 10 lines by clicking the 'load' button at once.If I want to see more context,it may take a lot of time to click the button.The button can be seen in my picture below.
![image](https://user-images.githubusercontent.com/76470592/146508468-7f2b9cab-469b-40d9-a8dc-e96038d641d2.png)

I wish the number of loading lines can be configured manually.

**Comment by ifrost** [comment]
Duplicate of [#43276](https://github.com/grafana/grafana/discussions/43276)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
